### DISCUSSION TITLE
Release Date of 4.0.1

### DISCUSSION THREAD

**Discussion by timoknapp** [osiegmar/logback-gelf#70] [Q&A] [answered]
Hi @osiegmar ,

do you know already when you will be able to release the latest version of `logback-gelf` including the latest version of logback (`1.2.8`). I already noticed, that you were updating the dependency in master branch ;)

Regards
Timo

**Comment by osiegmar** [comment]
Yes, also see https://github.com/osiegmar/logback-gelf/issues/67#issuecomment-993623157

I'm trying to release 4.0.1 since yesterday but Nexus was not able to publish the release so far. Their infrastructure is struggling with the current load as many projects are publishing (or trying to publish) new artifacts at the moment.

Please also note that (as per [LOGBACK-1591](https://jira.qos.ch/browse/LOGBACK-1591)) the problem doesn't seem that big as it only occurs if you use JNDI/JDBC in your configuration and the new Logback versions seems to simply disable (or deleted) the functionality.

**Comment by osiegmar** [comment]
Please also note, that logback-gelf 4.0.0 is compatible with Logback 1.2.8 without any changes. So, you don't need to wait for a 4.0.1 release, you could simply configure an additional logback 1.2.8 dependency.

**Reply by timoknapp** [reply]
Hi @osiegmar ,

thats exactly what I did already. But I'd still prefer using the most recent version of `logback-gelf` though :)

> So, you don't need to wait for a 4.0.1 release, you could simply configure an additional logback 1.2.8 dependency.

**Comment by osiegmar** [comment]
Nexus still tells me "Operation in progress" ... :-(

**Reply by timoknapp** [reply]
Yeah, today has been a nightmare. Almost any registry complaint about rate limit issues..

**Reply by osiegmar** [reply]
Still no progress on this. Also see https://status.maven.org

**Comment by osiegmar** [comment, marked as answer]
My account was now migrated to the new Nexus infrastructure and I was able to publish the release.

**Reply by timoknapp** [reply]
Thanks for the Update! That sounds like progress. So the release should be ready soon right?

**Reply by osiegmar** [reply]
Yes, probably in a few minutes (search index updates later).

**Reply by timoknapp** [reply]
Okay great, but I just noticed that the latest Release in GitHub is still 4.0.0. Are you going to update this too?

**Reply by osiegmar** [reply]
<img width="134" alt="image" src="https://user-images.githubusercontent.com/1918869/146585293-0a646477-6e96-4015-9b7e-6223cc23aa37.png">
